src/platform/errors/errorUtils.ts

```diff
@@ -551,6 +551,7 @@ function isBuiltInModuleOverwritten(
         return;
     }
 
+    // eslint-disable-next-line local-rules/dont-use-fspath
```

Comment:
This actually doesn't work without using fsPath

Reply:
Why not normalize the separators and always use `/` then we can use an includes or the like?
Then again, woudl'nt this code path crash in web, as `fsPath` is undefined!

Reply:
Oh yeah it would probably crash. Not used yet obviously. 

It will never include 'path' though because of the stupid '/' on the front. I guess I could account for that. 

I'd rather worry about this later when we actually get kernels working in the web extension.

Reply:
Cause before that we're going to switch to using the common vscode path stuff and try to use URIs everywhere.